OmiseSDK/Capability.swift

```diff
@@ -16,16 +16,19 @@ public struct Capability: Object {
         return backends[.card]
     }
 
-    public subscript(type: OMSSourceTypeValue) -> Capability.Backend? {
+    public subscript(type: SourceTypeValue) -> Capability.Backend? {
         return backends[.source(type)]
     }
 }
 
 extension Capability {
     public static func ~= (lhs: Capability, rhs: CreateSourceParameter) -> Bool {
         func backend(from capability: Capability, for payment: PaymentInformation) -> Backend? {
-            let paymentSourceType = OMSSourceTypeValue(payment.sourceType)
-            return capability[paymentSourceType]
+            if let paymentSourceType = SourceTypeValue(payment.sourceType) {
+                return capability[paymentSourceType]
+            } else {
+                return nil
+            }
```

Comment:
```suggestion
            }
            return nil
```

Reply:
Same as above, if there clear if/else scenario then most of the time this syntax is much faster to read and understand. And then if you would like to add more code there you have less chance place a code in a wrong place. Again in cases like that I prefer to have a clear vision of developer intension. Not always, but here it looks ok to me.